changelog/unreleased/enhancement-print-version

```diff
@@ -0,0 +1,6 @@
+Enhancement: Print version numbers
+
+The package version of the web UI and the version of the backend (if available) now get printed to the javascript console and get set as meta generator tag in the html head. This makes it possible to easily reference versions in bug reports.
```

Comment:
just a tiny one, no deal breaker...

`The package version of the web UI and the version of the backend (if available) now get printed to the browser console and get set as meta generator tag in the html head. This makes it possible to easily reference versions in bug reports.
`

Reply:
Thank you, changed it :-)